helpers.go

```diff
@@ -417,7 +417,10 @@ func NewUpdate(offset int) UpdateConfig {
 //
 // link is the url parsable link you wish to get the updates.
 func NewWebhook(link string) WebhookConfig {
-	u, _ := url.Parse(link)
+	u, err := url.Parse(link)
+	if err != nil {
+		log.Println("malformed URL: %s, error: %s", link, err)
```

Comment:
Should be `log.Printf()` not `log.Println()`

Reply:
agree. updated

Reply:
updated